pkg/radar/grouping.go

```diff
@@ -29,10 +29,10 @@ func (s *scope) findGroupForAircraft(trackfile *trackfiles.Trackfile) *group {
 //
 // - are within 3000 feet in altitude of each other
 //
-// These are tripled from the ATP numbers beacause the DCS AI isn't amazing at holding formation.
+// These are increased from the ATP numbers beacause the DCS AI isn't amazing at holding formation.
 // We allow mixed platform groups because these are fairly common in DCS.
 func (s *scope) addNearbyAircraftToGroup(this *trackfiles.Trackfile, group *group) {
-	spreadInterval := 3 * unit.NauticalMile
+	spreadInterval := 5 * unit.NauticalMile
```

Comment:
`5` should be defined as a constant somewhere, no?

Reply:
`3000` too actually, although 3000 shouldn't even be here right? groups are only range, not altitude. stacks (10,000) are for the fill ins.

although this is out of scope for the pr

Reply:
1. I'm preferring to store distances/altitudes as `unit.Length` rather than `int` since there's a lot of conversion between meters, nautical miles and feet in this code. `unit.Length` is mutable and cannot be used as a constant, only a var.

2. I could still make this a package-level var, but this is the only place this particular value is used (I broke from ACC and am instead tuning this based on the DCS AI's formation flight behavior). The `conf` package does contain some lengths that are reused.